scripts/demo/demo_benchmark_app.sh

```diff
@@ -122,6 +122,8 @@ elif [[ $DISTRO == "ubuntu" ]]; then
     system_ver=`cat /etc/lsb-release | grep -i "DISTRIB_RELEASE" | cut -d "=" -f2`
     if [ $system_ver = "18.04" ]; then
         sudo -E apt-get install -y libpng-dev
+    elif [ $system_ver = "20.04" ]; then
+        sudo -E apt-get install -y libpng-dev
     else
         sudo -E apt-get install -y libpng12-dev
```

Comment:
Lets refactor this section:
```
if [ $system_ver = "16.04" ]; then
    sudo -E apt-get install -y libpng12-dev
else
    # Ubuntu 18.04+
    sudo -E apt-get install -y libpng-dev
fi
```

Reply:
@alalek this will install libpng-dev for ubuntu 14 and other older versions. and we don't want that.

Reply:
I know, but these EOL versions are not supported here anymore (at least on master branch).

Reply:
@alalek created another PR ( #1827 ) with the fixes you suggested

Reply:
@ilya-lavrenov I have closed that PR and updated this one